enterprise/backend/src/metabase_enterprise/serialization/v2/load.clj

```diff
@@ -21,8 +21,15 @@
 
   This map works around this: given a model (e.g. `Card`) that triggered a dependency loop, it provides a set of keys
   to remove from the model so that we'll be able to successfully load it."
-  {"Dashboard" #{:dashcards}
-   "Card" #{:dashboard_id}})
+  {"Dashboard" #{[:dashcards :* :visualization_settings]}
+   "Card" #{[:dashboard_id]}})
+
+(defn- remove-by-path
```

Comment:
Desperately wanted to use a zipper here.

Reply:
Nice. Can you update the docstring in `model->circular-dependency-keys` to explain that it no longer `"provides a set of keys to remove from the model"`?